apps/web/src/app/(dynamicPages)/profile/[username]/permissions/_components/manage-keys-add-keys.tsx

```
@@ -1,67 +1,113 @@
-      <KeyInput ref={keyInputRef} />
+      {renderStepIndicator()}
 
-      <WalletSeedPhrase
-        size="sm"
-        showTitle={false}
-        username={activeUser?.username!}
-        onValidated={handleSaveKeys}
-      />
+      <div className="mt-4">
+        {currentStep === 1 && (
+          <Step1Authenticate username={activeUser?.username!} onNext={handleStep1Next} />
```

Biome (2.3.13)

[error] Forbidden non-null assertion after optional chaining.

Optional chaining already handles nullish values. Using non-null assertion defeats its purpose and may cause runtime errors.
Consider using the nullish coalescing operator `??` or optional chaining throughout the chain instead.

(lint/suspicious/noNonNullAssertedOptionalChain)